src/Shared/NodePacketTranslator.cs

```diff
@@ -207,10 +208,27 @@ public void Translate(ref string[] array)
                 }
             }
 
+            /// <inheritdoc />
+            public void Translate(ref HashSet<string> set)
+            {
+                if (!TranslateNullable(set))
+                {
+                    return;
+                }
+
+                int count = _reader.ReadInt32();
+                set = new HashSet<string>();
+
+                for (int i = 0; i < count; i++)
+                {
+                    set.Add(_reader.ReadString());
+                }
+            }
+
             /// <summary>
             /// Translates a list of strings
             /// </summary>
-            /// <param name="list">The list to be translated.</param>
+            /// <param name="list"></param>
```

Comment:
? indeed. No idea what happened, must have been the code elves stealing my comment :)